inscriptions/ISic1525.xml

```diff
@@ -170,11 +175,23 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
+                <ab>
+                    <lb n="1"/>Ἀρτεμισί<choice><corr>α</corr><sic>λ</sic></choice> Εὐ
+                    <lb n="2" break="no"/>τύχης ἠγόρ<supplied reason="omitted">ε</supplied>σαι 
+                    <lb n="3"/>ἰδίᾳ κλητρί τύμ
+                    <lb n="4" break="no"/>βον ἐβδομ<supplied reason="omitted">ή</supplied>κον
```

Comment:
num on the price?

Reply:
`<measure type="currency" unit="whatever-the-currency-here-is"><num value="number-value-here">...</num></measure>`